src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindInFilesDialog.java

```diff
@@ -104,6 +100,18 @@ private native JsArrayString getFilePatternsNative() /*-{
          return this.filePatterns;
       }-*/;
 
+      public final String[] getExcludeFilePatterns()
+      {
+         return JsUtil.toStringArray(getExcludeFilePatternsNative());
+      }
+
+      private native JsArrayString getExcludeFilePatternsNative() /*-{
+
+         if (!this.excludeFilePatterns)
+            this.excludeFilePatterns="";
```

Comment:
Should this be `[]` since the function returns an array?

Reply:
Fixed